app/budget/services/actions/budgetActions.js

```diff
@@ -39,6 +39,39 @@ class BudgetActions {
 						}
 					}
 
+					var sectionGroupCosts = results.sectionGroupCosts;
+					var terms = ["05", "07", "10", "01", "03"];
+
+					var subjectCode = results.courses[0].subjectCode;
+					var termCodes = terms.map(function(term) {
+						if (term == "01" || term == "03") {
+							return (parseInt(year) + 1) + term;
+						}
+						return year + term;
+					});
+
+					termCodes.forEach(function(termCode) {
+						DwService.getDwCensusData(subjectCode, null, termCode).then(function(census) {
+							// match courseNumber and TermCode and inject currentEnrollment number
+							const currentCensusSnapshot = census.filter(function(c) {
+								return c.snapshotCode == "CURRENT";
+							});
+
+							currentCensusSnapshot.forEach(function(courseCensus) {
+								sectionGroupCosts.forEach(function(sectionGroupCost) {
+									if (sectionGroupCost.courseNumber == courseCensus.courseNumber && sectionGroupCost.termCode == courseCensus.termCode) {
+										sectionGroupCost.currentEnrollment
```

Comment:
Did prettier do this? The indentation looks odd

Reply:
Yeah, pretty sure that was due to the formatter, I actually undid the format but not sure why that part stuck. 

Any preference for keeping it one-line or multi-line? Or even re-write it without the ternary?

```js
    sectionGroupCost.currentEnrollment ?
      (sectionGroupCost.currentEnrollment += courseCensus.currentEnrolledCount) :
      sectionGroupCost.currentEnrollment = courseCensus.currentEnrolledCount;
```

Reply:
Ternary looks ok, logic is fairly simple